windowsforms/matching-game/readme.md

```diff
@@ -1,6 +1,6 @@
 # Matching Game Sample
 
-These two samples are used by the [How to migrate a Windows Forms desktop app to .NET 5](https://docs.microsoft.com/dotnet/desktop/winforms/migration/?view=netdesktop-5.0&preserve-view=true) article.
+These two samples are used by the [.NET Upgrade Assistant](https://docs.microsoft.com/dotnet/core/porting/upgrade-assistant-wpf-framework) article.
```

Comment:
Presumably this is a Windows Forms sample, and the original docs pointed to Windows Forms related docs. Now it points to WPF related docs...